### ISSUE TITLE
1.12 Planning

### ISSUE THREAD

**Issue by JonL98** [ata4/dragon-mounts#51]
Initially only three errors when compiling with 1.12, two of which are the same issue.
![screenshot 1](https://user-images.githubusercontent.com/19556821/30175570-d04ab2c0-93b3-11e7-827d-58187c42f1d0.png)

**Comment by ghost** [comment]
BufferBuilder can be used instead of VertexBuffer. About RuntimeOptionGuiHandler I don't know.

**Comment by JonL98** [comment]
No, VertexBuffer just changed locations. Still not sure what the RuntimeOptionGuiHandler issue is all about. I don't think any of DragonMountsConfigGuiFactory's dependencies changed for 1.12. 
Any insights, @ata4 ?

**Comment by Sunconure11** [comment]
@JonL98 I'm going to take a look at your fork, and see about fixing it. I ported a few mob-related mods and figured out some things.

Note that I'll likely change the mappings, but do not fret.

Edit 1: I think you need to put in whatever registry hackery you did. I set it up, and got it running, but it gives me this error:

https://pastebin.com/wCwf5K2G

This is my fork, BTW:

https://github.com/Sunconure11/dragon-mounts/tree/1.12.2
